### PR TITLE
aliases: Don't overshadow fd

### PR DESCRIPTION
**Pull request by HaoZeke**
## Standards checklist:

- [x] The PR title is descriptive.
- [x] The PR doesn't replicate another PR which is already open.
- [x] I have read the contribution guide and followed all the instructions.
- [x] The code follows the code style guide detailed in the wiki.
- [x] The code is mine or it's from somewhere with an MIT-compatible license.
- [x] The code is efficient, to the best of my ability, and does not waste computer resources.
- [x] The code is stable and I have tested it myself, to the best of my abilities.

## Changes:

- Checks for fd

## Other comments:

...

### CODE REVIEW

**Review by 53jk1 — APPROVED** [review]

**Comment by cascadingstyletrees** [comment]
Any chance this could be merged?

**Comment by cascadingstyletrees** [comment]
unset `fd` to the end of my `.zshrc` in the meantime.

File: plugins/common-aliases/common-aliases.plugin.zsh [hunk 1]
```diff
@@ -34,7 +34,9 @@ alias -g P="2>&1| pygmentize -l pytb"
 
 alias dud='du -d 1 -h'
 alias duf='du -sh *'
-alias fd='find . -type d -name'
+if ! which fd >/dev/null 2>%1; then
+  alias fd='find . -type d -name'
+fi
```
Comment (mcornella) [inline plugins/common-aliases/common-aliases.plugin.zsh:39]: Syntax error, it should be `&1`. That said, it's more idiomatic to use `(( $+commands[fd] ))`:
```suggestion
(( $+commands[fd] )) || alias fd='find . -type d -name'
```

Comment (HaoZeke) [reply plugins/common-aliases/common-aliases.plugin.zsh]: Thanks!!

**Review by mcornella — APPROVED** [review]

**Pull request merged**